version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.4.22",
+  "version": "1.4.23",
```

Comment:
>23 [](http://example.com/codeflow?start=18&length=2)

btw you don't actually need to publish this package to get this change, just update your submodule for now!